Code/GraphMol/MolDraw2D/MolDraw2D.cpp

```diff
@@ -323,8 +358,15 @@ Point2D MolDraw2D::getDrawCoords(int at_num) const {
 // ****************************************************************************
 Point2D MolDraw2D::getAtomCoords(const pair<int, int> &screen_cds) const {
   int x = int(double(screen_cds.first) / scale_ + x_min_ - x_trans_);
-  int y = int(double(screen_cds.second) / scale_ + y_min_ - y_trans_);
+  double y =
```

Comment:
I understand the int(double(...)) part, but I don't understand the double y = int(double(...)) part